octohook/models.py

```diff
@@ -253,6 +253,10 @@ class Repository(BaseGithubModel):
     public: Optional[bool]
     master_branch: Optional[str]
     permissions: Optional[Permissions]
+    allow_forking: Optional[bool]
+    is_template: Optional[bool]
+    topics: List[str]
```

Comment:
This is optional in the `__init__` block.

Reply:
I have it default to an empty list, but I changed it to just `payload.get("topics").